package.json

```diff
@@ -39,5 +39,10 @@
     "singleQuote": true,
     "trailingComma": "es5"
   },
-  "dependencies": {}
+  "dependencies": {
```

Comment:
these should be `devDependencies`, but more importantly I don't understand why they are needed. What happens if you remove them?

Reply:
I'm happy to rename to devDependencies.

It seems that this `package.json` and the `yarn.lock` file are used to identify dependencies to be pulled in. My workflow is: `rm -rf node_modules; yarn install; yarn build; yarn test`.

#### Without `"jest": "^29.7.0"` I get an error:

```
There was an error while running the Foam suite Error: ● Validation Error:

  Preset ts-jest not found.

  Configuration Documentation:
  https://jestjs.io/docs/configuration
```

and the tests do not run. I've moved this to devDependencies in the latest version of this pull request.

#### Previously, I would get different errors without the others, but now they do not produce errors.

Because it does not seem to be necessary to assume them, I've removed the others.